app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb

```diff
@@ -6,10 +6,12 @@ def parse
 
   def configuration_profiles
     collector.templates.each do |template|
+      target_platform = template["manifest"]["template_provider"] if defined? template["manifest"]
```

Comment:
You can do `template.dig("manifest", "template_provider")` if you're worried about `template["manifest"]` being nil

Reply:
Nice, thanks for the tip